guides/common/modules/con_using-ipxe-to-reduce-provisioning-times.adoc

```diff
@@ -0,0 +1,6 @@
+[id="using-ipxe-to-reduce-provisioning-times_{context}"]
+= Using iPXE to reduce provisioning times
+
+iPXE is an open-source network-boot firmware.
+It provides a full PXE implementation enhanced with additional features, such as booting from an HTTP server.
+For more information, see https://ipxe.org[ipxe.org].
```

Comment:
Re: support from Red Hat:

* iPXE in RHEL is only supported in the libvirt/qemu environment. Bare-metal PXE or HTTP protocols are not supported.
* The iPXE binary shipped in RHEL is compiled without TLS and HTTPS support.